### ISSUE TITLE
[Keyboard Navigation-Microbit Feedback-Give Feedback] 'Report a problem' button is not accesible using keyboard

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2549]
*User Experience*
A disable user accustomed to using the keyboard for navigating through the UI elements on the system, will find it difficult to not be able to access all interactive controls on call a function pane via keyboard.

*Test Environment *
OS: RS2 Version 1703(OS Build 15063.2
Platform: Edge.

*Repro Steps*
1-Navigate to https://makecode.microbit.org/acc 
2-Navigate to the Microbit section element and select Code control given on it
3-Navigate to the Settings control lying in the header section on the page and select it
4-Navigate to the Give feedback menu option listed under the Settings menu control and select it
5-Navigate to the various controls lying in the pane opened.
6-Verify 'Report a problem' button is accessible using keyboard..

*Actual Result *
'Report a problem' button is not accessible using keyboard

*Expected Result 
'Report a problem' button must be accessible using keyboard

*MAS Reference*
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

*Suggested fix*
1. Always ensure each functionality of a page is keyboard accessible. E.g. A page with images used as links changes when the user hovers over the image with a mouse. To provide keyboard users with a similar experience, the image is also changed when a user tabs to it
2. Always try to use standard HTML form controls and link elements to provide keyboard operation and assistive technology interoperability of interactive user interface elements.
3. Use of both keyboard-specific and mouse-specific events with code that has a scripting function associated with an event. 
Use mousedown with  keydown 
Use mouseup  with keyup 
Use click   with  keypress 
Use mouseover with  focus 
Use mouseout with blur 
e.g.< img onclick="nextPage();" onkeypress="nextPage();" tabindex="0" src="arrow.gif" alt="Go to next page" >Key 
References for Fix:
http://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.htmlPlease 

Refer attachment file
[MAS 2.1.1_Feedback_ReportProblemButtonNotAccesible.zip](https://github.com/Microsoft/pxt/files/1179870/MAS.2.1.1_Feedback_ReportProblemButtonNotAccesible.zip)

**Comment by veler** [comment]
The give feedback website is an external website. We don't have the control on it. We cannot fix it.

**Comment by veler** [comment]
Needs to be forwarded to [Survey Monkey](https://www.surveymonkey.com).

**Comment by abchatra** [comment]
External issue with Survey Monkey.
